src/js/components/TextInput/doc.js

```diff
@@ -74,6 +74,7 @@ The suggestion contains the object chosen from the supplied suggestions.`,
     placeholder: PropTypes.oneOfType([
       PropTypes.string,
       PropTypes.element,
+      PropTypes.node,
```

Comment:
I don't think we should add `node` if it doesn't contribute to solving the issue.